config/locales/views/index/en.yml

```diff
@@ -0,0 +1,4 @@
+en:
```

Comment:
I am not in favor if this path convention is the original approach. Do you mean the following path `config/locales/views/application/en.yml` instead of `config/locales/views/index/en.yml`? I know that you have used the `config/locales/views/application/en.yml` path, but this is a method from `ApplicationController`.

In views folder I was expecting an entire file for each folder (almost each controller) like the following (taking user folder as example):
```yml
# config/locales/views/user/en.yml
en:
  views:
    users:
      # /users/dashboard
      dashboard:
        message: ...
      id:
        # /users/:id
        show:
          message: ...
        # /users/:id/edit
        edit:
          message: ...
```

What do you think?